src/util/authUtils.js

```diff
@@ -83,3 +83,21 @@ export const applyServerState = (server, options) => {
 	server.state('refresh_token', authCookieOptions);
 };
 
+export const assignMemberState = options => (request, reply) => {
+	const memberValue = options.API_HOST.includes('.dev.') ?
+		request.state.MEETUP_MEMBER_DEV :
+		request.state.MEETUP_MEMBER;
+	request.state.MEETUP_MEMBER = memberValue;
+	request.state.MEETUP_MEMBER_DEV = memberValue;
```

Comment:
this is adapting to the setup we have in Meetup Classic, where you get MEETUP_MEMBER_DEV from www.dev.meetup.com, and MEETUP_MEMBER from www.meetup.com - I've tried to 'normalize' that setup here so that you can just reference `MEETUP_MEMBER` throughout the rest of the code, instead of having something like `if (C.IS_DEV) { MEETUP_MEMBER_DEV...` all over the show.

One thing that we might want to talk about is whether we should restrict dev to pointing to `www.api.dev.meetup.com`, the way that Meetup Classic dev works, rather than letting devs set the `API_HOST` env variable themselves - there's some complexity/constraints either way, but one of the costs of keeping it flexible is this 'normalizing' code to apply the right MEETUP_MEMBER cookie value.